src/material/chips/_chips-theme.scss

```diff
@@ -19,7 +19,14 @@ $mat-chip-remove-font-size: 18px;
   @include mat-chips-color(mat-color($palette, default-contrast), mat-color($palette));
 
   .mat-ripple-element {
-    background: mat-color($palette, default-contrast, 0.1);
+    $ripple-color: mat-color($palette, default-contrast);
```

Comment:
shouldn't we pass the `0.1` opacity to `mat-color`? Unless I miss something, it looks like we only apply opacity to the color when CSS variables are used right now?

Reply:
Ops, good catch. I think I had it correctly and then I moved something around without testing and it broke.